packages/flutter_markdown/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.6.17+4
+
+* Fixes an incorrect behavior of code block decoration.
```

Comment:
I would prefer if this was a little more specific of what the "incorrect behavior" is. If I was looking for why something had changed with the clip behavior, this might not catch my attention.

Reply:
Thanks, corrected.